tests/makefile.in

```diff
@@ -108,7 +108,7 @@ core-post:
 
 .NOTPARALLEL:
 
-check: core test1 test2 test3 test4 test5 test6 test7 test8 test9 test10 test11
+check: core test1 test2 test4 test6 test7 test8 test9 test10 test11
```

Comment:
Can we put back `test3` and `test5` now?  Without them `make check`  takes
```
23.011u 3.901s 0:26.97 99.7%	0+0k 0+20144io 0pf+0w
```
on my 10-year-old Linux box and with them it takes
```
25.382u 4.370s 0:29.94 99.3%	0+0k 0+23288io 0pf+0w
```